src/mbgl/style/paint_property_binder.hpp

```diff
@@ -136,6 +139,8 @@ class SourceFunctionPaintPropertyBinder : public PaintPropertyBinder<T, A> {
     }
 
     void populateVertexVector(const GeometryTileFeature& feature, std::size_t length) override {
+        auto evaluated = function.evaluate(feature, defaultValue);
+        this->statistics.add(evaluated);
         auto value = attributeValue(function.evaluate(feature, defaultValue));
```

Comment:
I accidentally reverted this line in my changes, should be `auto value = attributeValue(evaluated);`